.travis.yml

```diff
@@ -3,7 +3,7 @@ language: java
 install: mvn install -DskipTests=true
 script: mvn test
 jdk:
-  - oraclejdk8
+  - oraclejdk9
```

Comment:
I would not replace it but add it: Travis should be able to run the tests using both the JDKs so we verify it stays compatible with JDK8 and JDK9 at the same time

Reply:
How would I do that?

Reply:
I think you can just add the second line without removing the first one

Reply:
It worked :-O